data/templates/js/collapse-infotable.js

```diff
@@ -1,4 +1,10 @@
 $(function() {
+        // Make sure we only have one div for ordering of the elements
+        var divP = $(".mw-parser-output");
+        if(divP.parent().is("div")){
```

Comment:
We are missing spaces after _if_ and before curly bracket

Reply:
Fixed.